src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
I use the standard QGC FlightGear configuration and I would agree that the default noise levels are a little high.  I've halved all the values, except for roll/pitch/yaw rate, on my local build environment and they now give results that mirror real flight data quite well.  

I agree with @LorenzMeier that a KISS approach for noise values is what most users will want.  Though I'd really like to see a ui widget for being able to set these values using the setter functions in UAS.h:  https://github.com/mavlink/qgroundcontrol/blob/master/src/uas/UAS.h#L330-L381